fcsparser/api.py

```diff
@@ -134,7 +149,7 @@ def from_data(cls, data):
             obj.load_file(file_handle)
         return obj
 
-    def read_header(self, file_handle):
+    def read_header(self, file_handle, nextdata=0):
```

Comment:
Could update the doc-string to document `nextdata`? 

Also is there a better variable name than `nextdata` I think the `next` part is a bit confusing. 

Does this variable have the same meaning as `dataset` ? (If so perhaps we could use the same name?)

Reply:
nextdata directly refers to the $NEXTDATA text field, which specifies the byte offset of the header specifying the next data set. (yes, fcs data sets are completely separate for some reason, they can even be of different fcs versions inside the same file)

I have renamed it to nextdata_offset, so that the meaning of this variable is a bit clearer. We need this, because the offsets specified in the header of each data set is only relative to the start of the respective header.